Android/PushTutorial/push-tutorial-two/build.gradle

```diff
@@ -0,0 +1,47 @@
+apply plugin: 'com.android.application'
```

Comment:
I'm not sure if we should have a series of step commits to build this app in the same way as the first - or it was the intention to build the whole app in a single step